ibis/backends/dask/execution/join.py

```diff
@@ -0,0 +1,109 @@
+import operator
+
+import dask.dataframe as dd
+from pandas import Timedelta
+
+import ibis.expr.operations as ops
+import ibis.util
+from ibis.backends.pandas.execution.join import (
+    _compute_join_column,
+    _extract_predicate_names,
+    _validate_columns,
+    execute_node,
+)
+
+from ..execution import constants
+
+
+@execute_node.register(
+    ops.AsOfJoin, dd.DataFrame, dd.DataFrame, (Timedelta, type(None))
+)
+def execute_asof_join(op, left, right, tolerance, **kwargs):
+    overlapping_columns = frozenset(left.columns) & frozenset(right.columns)
+    left_on, right_on = _extract_predicate_names(op.predicates)
+    left_by, right_by = _extract_predicate_names(op.by)
+    _validate_columns(
+        overlapping_columns, left_on, right_on, left_by, right_by
+    )
+
+    return dd.merge_asof(
```

Comment:
Ok cool! Didn't know this is supported.

Reply:
I don't think it is listed in the docs, but exists as part of https://docs.dask.org/en/latest/_modules/dask/dataframe/multi.html.